src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_NUMERIC = 1;
```

Comment:
```suggestion
    private static final int INDEX_OF_PRODUCT_COUNT = 1; 
```
let's try to use a more informative name